packages/stryker-mutator-specification/src/BlockMutatorSpec.ts

```diff
@@ -0,0 +1,33 @@
+import { expect } from 'chai';
+import ExpectMutation from './ExpectMutation';
+
+export default function BlockMutatorSpec(name: string, expectMutation: ExpectMutation) {
+
+  describe('BlockMutator', () => {
+
+    it('should have name "Block"', () => {
+      expect(name).eq('Block');
+    });
+
+    it('should mutate the block of a function into an empty block', () => {
+      expectMutation('function() { return 4; }', 'function() {}');
+    });
+
+    it('should mutate a single block', () => {
+      expectMutation('const a = 3; { const b = a; }', 'const a = 3; {}');
```

Comment:
Does every language that we (plan) to support with this spec have the `const` keyword?

Reply:
Yep, JSX, javascript and TypeScript support it. If it becomes a problem we can change it later